pages/osx/xsand.md

```diff
@@ -0,0 +1,5 @@
+# xsand
+
+> Xsan file system management daemon.
+> Provides services for the Xsan file system.
+> It should not be invoked manually.
```

Comment:
I'm really not happy about that link but there is hardly any official documentation

```suggestion
> Xsan file system management daemon. Provides services for the Xsan file system.
> It should not be invoked manually.
> More information: <https://developer.apple.com/support/downloads/Xsan-Management-Guide.pdf>.

- Start the daemon:

 `xsand`
```